Packs/Devo/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### Devo v2
+- Added fetch Alerts/Incidents pull limit option to prevent plugin from crashing.
+- Added fetch Alerts/Incidents pull from custom Devo tables.
```

Comment:
These changes were already made in version [1_2_0](https://github.com/demisto/content/pull/24747/files#diff-f31cb4e8e2f9ee5c5b4d6cb9547caf17fd089b91e1a2f2862fd4a1b5d8c43e77)